File: src/sys-base/src/main/java/com/wcpdoc/base/entity/RegistUser.java
package com.wcpdoc.base.entity;

import java.util.Date;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;

import lombok.Builder;
import lombok.Data;

/**
 * 注册用户实体
 * 
 * v1.0 zhanghc 2025年12月1日下午4:15:06
 */
@Data
@TableName(value = "SYS_REGIST_USER", autoResultMap = true)
@Builder
public class RegistUser {
	@TableId(type = IdType.AUTO)
	private Integer id;
	private String name;
	private String loginName;
	private String pwd;
	private Date registTime;
	private Integer orgId;
	private Integer state;
	private String remark;
	private Integer updateUserId;
	private Date updateTime;
}

File: h5/src/ts/base/registUser.ts
export interface RegistUser {
    id: number | null// ID
    name: string // 姓名
    loginName: string // 登录账号
    orgId: number | null // 机构ID
    registTime: string // 注册时间
    state: number | null// 状态（1：通过；2：拒绝；3：待审核）
    remark: string // 审批意见
}